pkgs/applications/networking/taler/default.nix

```
@@ -19,12 +19,8 @@ in rec {
     src = fetchgit {
       url = "https://git.taler.net/exchange.git";
       rev = "v${version}";
-      # REMOVEME: this should only be a problem for specifically v0.9.3
-      # When fetching submodules without deep clone we get the following error:
-      # "Server does not allow request for unadvertised object"
-      deepClone = true;
       fetchSubmodules = true;
-      sha256 = "sha256-txWwW5vqTblNgVIXdDkpNNZOXpY0udMaz4Wog1GobzE=";
+      sha256 = "sha256-QaddVnKxsGTQcpMCVA4vvsuSqjDThvB7izKMeWWrbO8=";
```

```suggestion
      sha256 = "";
```

Can you get the hash again? It shouldn’t be the same without `deepClone`